src/blocks/scratch3_pen.js

```diff
@@ -0,0 +1,231 @@
+var Cast = require('../util/cast');
+var Color = require('../util/color');
+var MathUtil = require('../util/math-util');
+var RenderedTarget = require('../sprites/rendered-target');
+
+// Place the pen layer in front of the backdrop but behind everything else
+// We should probably handle this somewhere else... somewhere central that knows about pen, backdrop, video, etc.
+// Maybe it should be in the GUI?
+var penOrder = 1;
```

Comment:
We have generally been handling things like this and the `stateKey` (below) like this:
https://github.com/LLK/scratch-vm/blob/develop/src/engine/sequencer.js#L23